### PR TITLE
Add graffiti example

### PR DESCRIPTION
**Pull request by lincolnfrog**

### CODE REVIEW

File: examples/graffiti.html [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+<script>
+
+var vrDisplay;
+var vrControls;
+var arView;
+
+var canvas;
+var camera;
+var scene;
+var renderer;;
```
Comment (ZohaibAhmed) [inline examples/graffiti.html]: 2 semicolons here

**Review by jsantell — APPROVED** [review]
Looks great! I'd copy over the styles in the other examples so we have the same "three.ar.js" and instruction blurbs as the other examples (see 225ffa378dd6856e0d3251cca2905f51e985b834 for the text selection fix), and you have the ARView.render fix from 47cfea207452fa6845cdf028303b6c5775a1d1b3 so I think this looks good to go

File: examples/graffiti.html (hunk 1 above)
Comment (jsantell) [reply examples/graffiti.html]: good eye!

**Comment by judax** [comment]
I have been testing it and I am seeing some weird behaviour. More testing is needed IMO (both iOS and Android).

File: examples/graffiti.html [hunk 2]
```diff
@@ -0,0 +1,433 @@
...
+  return {
+    position: new THREE.Vector3(position.x, position.y, position.z),
+    normal: new THREE.Vector3(normal.x, normal.y, normal.z),
+    velocity: new THREE.Vector3(velocity.x, velocity.y, velocity.z),
+    width: strokeWidth
+  };
+}
+
+function processDrawBrush() {
+  // Add the draw point to the current stroke.
```
Comment (lincolnfrog) [inline examples/graffiti.html]: Not sure why but this code doesn't work. Brush/reticle is very glitchy. Something wrong here.

Comment (rezaali) [reply examples/graffiti.html]: I believe the newest commit should resolve the glitchy nature of the brush, it essentially hides it if there is no device motion.

File: examples/graffiti.html [hunk 3]
```diff
@@ -0,0 +1,540 @@
...
+  geometry.addAttribute('velocity', new THREE.BufferAttribute(velocities, 3).onUpload(disposeArray));
+  geometry.computeBoundingSphere();
+
+  var mesh = new THREE.Mesh(geometry, drawMaterial);
+  mesh.drawMode = THREE.TriangleStripDrawMode;
+  return mesh;
+}
+
+var accelerationArray = [];
+function checkForShake()
```
Comment (lincolnfrog) [inline examples/graffiti.html]: Couple comments?

File: examples/graffiti.html [hunk 4]
```diff
@@ -0,0 +1,540 @@
...
+  if(b0.position.distanceTo(b1.position) < MINIMIM_POINT_DISTANCE) {
+    return;
+  }
+
+  brush = generateStroke(brushStroke);
+  scene.add(brush);
+}
+
+
+function generateStroke(strokeData) {
```
Comment (lincolnfrog) [inline examples/graffiti.html]: Couple comments

**Review by lincolnfrog — COMMENTED** [review]
Looks good!

**Pull request merged**